modules/administration-guide/partials/proc_importing-extensions-that-are-not-in-the-open-vsx-registry.adoc

```diff
@@ -3,14 +3,53 @@
 [id="importing-extensions-that-are-not-in-the-open-vsx-registry"]
 = Importing extensions that are not in the Open VSX Registry
 
-In the intro, please clarify if this procedure is about adding extensions to Che's embedded Open VSX Registry.
+This section describes how to add VS Code extensions that are not in the link:https://open-vsx.org/[Open VSX Registry] to {prod-short}'s embedded Open VSX Registry.
 
-Please clarify the use cases for adding extensions that are not in the Open VSX Registry. Is this procedure about adding extensions that are in https://marketplace.visualstudio.com/VSCode? If so, then please clearly state it as the (main?) use case for this procedure.
+It's possible that link:https://open-vsx.org/[Open VSX Registry] doesn't contain an extension that needs to be used in the workspace. If the extension was not yet published on the link:https://open-vsx.org/[Open VSX Registry]. In that case, please reach out to the extension's author and ask them to publish the extension on the Open VSX registry, the link:https://open-vsx.org/[how to Publish an Extension] docs provide instructions or try to import it into the local registry by using the procedure from this page.
 
 .Prerequisites
-
-?
+. The vsix file of the VS Code extension should be deployed somewhere to download it by the direct link.
+. The extenshion's id and version that is going to be added.  
 
 .Procedure
-
-?
+. Modify `openvsx-sync.json` file in the link:https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json[plugin-registry repository] to add an information about the extension:
```

Comment:
```suggestion
. Modify the plug-in registry's link:https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json[`openvsx-sync.json`] file to add an extension:
```

Reply:
Note that this is a link to the downstream file... we probably need a new antora variable for this URL so we can pull either https://github.com/eclipse-che/che-plugin-registry/blob/main/openvsx-sync.json or https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json. 

Or should we link to the raw file instead?

https://raw.githubusercontent.com/eclipse-che/che-plugin-registry/main/openvsx-sync.json
vs.
https://raw.githubusercontent.com/redhat-developer/devspaces/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json